docs/ref/request-response.txt

```diff
@@ -425,20 +425,24 @@ Methods
     Returns ``True`` if the request is secure; that is, if it was made with
     HTTPS.
 
-.. method:: HttpRequest.accepts(mime_type)
+.. method:: HttpRequest.get_preferred_type(media_types)
 
-    Returns ``True`` if the request ``Accept`` header matches the ``mime_type``
-    argument:
+    .. versionadded:: 5.2
+
+    Returns the most preferred mime type from ``media_types`` which the client
+    accepts, according to the ``Accept`` header.
+
+    Assuming the client sends an ``Accept`` header of
+    ``text/html,application/json;q=0.8``:
 
     .. code-block:: pycon
 
-        >>> request.accepts("text/html")
-        True
+        >>> request.get_preferred_type(["text/html", "application/json"])
+        "text/html"
+        >>> request.get_preferred_type(["application/json", "text/plain"])
+        "application/json"
```

Comment:
Reading this I'm left wondering, so could you add an example for a non-matching list, so as... 

    >>> request.get_preferred_type(["application/xml", "text/plain"])
    None

(Probably worth a sentence.)

Users can request what they like. I've got what I've got 😅